Source/IGListBindingSectionController.m

```diff
@@ -54,7 +54,11 @@ - (void)updateAnimated:(BOOL)animated completion:(void (^)(BOOL))completion {
         }
         
         oldViewModels = self.viewModels;
-        self.viewModels = [self.dataSource sectionController:self viewModelsForObject:self.object];
+
+        id<IGListDiffable> object = self.object;
+        IGAssert(object != nil, @"Expected IGListBindingSectionController object to be non-nil before updating.");
+        
+        self.viewModels = [self.dataSource sectionController:self viewModelsForObject:object];
```

Comment:
This was added because the compiler (correctly) complained about the nullability mismatch.